### PR TITLE
use uuid as key for aether-subscriber

### PR DESCRIPTION
**Pull request by sbconsulting**
Left version 1.0.0 models in place in case we want to reference them again some day. The primary changes to the version 2.0.0 models are:
   * renamed "ueid" to "id" and set the type of uuid
   * structured imsi as a choice between wildcard and range, and broke out the range as two uint64.
   * added display-name to all models
Note that the example gnmi using string_val instead of int_val is intended and is making use of one bug (strings are implicitly converted to integers if they contain digits) to work around a different bug (large int_val get converted to floats instead of int64).

### CODE REVIEW

File: modelplugin/aether-2.0.0/aether.tree [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        +--rw display-name?   string
+        +--rw type?           string
+        +--rw filter?         string
+        +--rw description?    string
+module: aether-subscriber
+  +--rw subscriber
+     +--rw ue* [id]
+        +--rw id                       ietf:uuid
+        +--rw display-name?            string
+        +--rw (imsi)?
```
Comment (SeanCondon) [inline modelplugin/aether-2.0.0/aether.tree:14]: I think we did say in the Monday call that there would be a tree to handle each access technology. So there might be a "5G" or "ran" container here before you get in to IMSI & plmn etc

**Review by SeanCondon — APPROVED** [review]
One minor comment from me - can be addressed later if necessary

File: modelplugin/aether-2.0.0/aether.tree (hunk 1 above)
Comment (sbconsulting) [reply modelplugin/aether-2.0.0/aether.tree:14]: I was thinking about deferring this until we have more information -- my concern is that there may be overlap between the multiple uses and I don't want to repeat the same piece of information in two or more subtrees. Let's take this in stages, we'll look at what information there is in SD-Ran next. Then we also need to look at what keys will be added for 5G that aren't part of 4G.

Comment (SeanCondon) [reply modelplugin/aether-2.0.0/aether.tree:14]: sounds good - we can tackle it later then

**Pull request merged**